### PR TITLE
GPIB-450

### PR DESCRIPTION
**Pull request by SharmaDinesh**

### CODE REVIEW

File: src/components/deposit-hints/DepositHintsForm.js [hunk 1]
```diff
@@ -32,6 +39,9 @@ const DepositHintsForm = ({
   enterprise,
   sourceFrom
 }) => {
+  // const abc = (e) => {
+  //   const re = /^[0-9\b]+$/;
+  // };
```
Comment (rellfy) [inline src/components/deposit-hints/DepositHintsForm.js]:
```suggestion
```

**Review by rellfy — CHANGES_REQUESTED** [review]
Just a few lines that can be removed

File: src/components/deposit-hints/DepositHintsForm.js (hunk 1 above)
Comment (SharmaDinesh) [reply src/components/deposit-hints/DepositHintsForm.js]: Done

**Review by ahinchliff — APPROVED** [review]

File: src/components/deposit-hints/DepositHintsForm.js [hunk 2]
```diff
@@ -6,13 +6,20 @@ import SubmitSpinnerButton from "components/forms/SubmitSpinnerButton";
...
   if (!values.employerName) errors.employerName = requiredMsg;
-  if (!values.depositAmount && String(values.depositAmount) !== "0")
-    errors.depositAmount = requiredMsg;
+  if (values.depositAmount === "" || re.test(values.depositAmount)) {
+    if (!values.depositAmount && String(values.depositAmount) !== "0")
+      errors.depositAmount = requiredMsg;
+  } else {
+    errors.depositAmount = depositAmountMsg;
+  }
+
```
Comment (xiangzhe0812) [inline src/components/deposit-hints/DepositHintsForm.js]: if(!values.depositAmount) errors.depositAmount = requireMsg;
if(values.depositAmount && !re.test(values.depositAmount))  errors.depositAmount = depositAmountMsg

**Review by xiangzhe0812 — COMMENTED** [review]
one little change

File: src/components/deposit-hints/DepositHintsForm.js (hunk 2 above)
Comment (xiangzhe0812) [reply src/components/deposit-hints/DepositHintsForm.js]: wo can do like above

Comment (SharmaDinesh) [reply src/components/deposit-hints/DepositHintsForm.js]: Done

**Review by xiangzhe0812 — APPROVED** [review]

**Pull request merged**